/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `PYTORCH_PRETRAINED_BERT_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `PYTORCH_TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/transformers/utils/hub.py:106: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
/storage/weinig/cdl-inflation-narrative/annotation/labelstudio/agreement.py:219: FutureWarning: Downcasting behavior in `replace` is deprecated and will be removed in a future version. To retain the old behavior, explicitly call `result.infer_objects(copy=False)`. To opt-in to the future behavior, set `pd.set_option('future.no_silent_downcasting', True)`
  train['label'] = train['label'].replace(self.label2id_map)
/storage/weinig/cdl-inflation-narrative/annotation/labelstudio/agreement.py:220: FutureWarning: Downcasting behavior in `replace` is deprecated and will be removed in a future version. To retain the old behavior, explicitly call `result.infer_objects(copy=False)`. To opt-in to the future behavior, set `pd.set_option('future.no_silent_downcasting', True)`
  valid['label'] = valid['label'].replace(self.label2id_map)
/storage/weinig/cdl-inflation-narrative/annotation/labelstudio/agreement.py:221: FutureWarning: Downcasting behavior in `replace` is deprecated and will be removed in a future version. To retain the old behavior, explicitly call `result.infer_objects(copy=False)`. To opt-in to the future behavior, set `pd.set_option('future.no_silent_downcasting', True)`
  test['label'] = test['label'].replace(self.label2id_map)
Some weights of DistilBertForSequenceClassification were not initialized from the model checkpoint at distilbert/distilbert-base-uncased and are newly initialized: ['classifier.bias', 'classifier.weight', 'pre_classifier.bias', 'pre_classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
annotator 5: 0 document(s) to annotate.
annotator 7: 0 document(s) to annotate.
annotator 8: 0 document(s) to annotate.
annotator 9: 0 document(s) to annotate.
inner_id_list 488, annotator_5: 488
Updated annotation from user 5, result saved in ./export/task_1_annotation.csv
Task 1 (User 5 - Sviatoslav)
inner_id_list 488, annotator_7: 488
Updated annotation from user 7, result saved in ./export/task_1_annotation.csv
Task 1 (User 6 - Julia)
inner_id_list 488, annotator_8: 488
Updated annotation from user 8, result saved in ./export/task_1_annotation.csv
Task 1 (User 7 - Darian)
inner_id_list 488, annotator_9: 488
Updated annotation from user 9, result saved in ./export/task_1_annotation.csv
Task 1 (User 8 - Kim)
has winner ratio: 0.8790983606557377
all agreeing count: 232
0.6858854912630578
Map:   0%|          | 0/341 [00:00<?, ? examples/s]Map: 100%|██████████| 341/341 [00:00<00:00, 578.92 examples/s]Map: 100%|██████████| 341/341 [00:00<00:00, 577.15 examples/s]
Map:   0%|          | 0/49 [00:00<?, ? examples/s]Map: 100%|██████████| 49/49 [00:00<00:00, 523.95 examples/s]
Map:   0%|          | 0/98 [00:00<?, ? examples/s]Map: 100%|██████████| 98/98 [00:00<00:00, 611.93 examples/s]Map: 100%|██████████| 98/98 [00:00<00:00, 608.13 examples/s]
Traceback (most recent call last):
  File "/storage/weinig/cdl-inflation-narrative/annotation/labelstudio/agreement.py", line 303, in <module>
    inflation_narrative.train_sequence_classifier()
  File "/storage/weinig/cdl-inflation-narrative/annotation/labelstudio/agreement.py", line 260, in train_sequence_classifier
    metric = evaluate.load("f1")
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/loading.py", line 748, in load
    evaluation_module = evaluation_module_factory(
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/loading.py", line 680, in evaluation_module_factory
    raise e1 from None
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/loading.py", line 639, in evaluation_module_factory
    ).get_module()
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/loading.py", line 479, in get_module
    local_path = self.download_loading_script(revision)
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/loading.py", line 469, in download_loading_script
    return cached_path(file_path, download_config=download_config)
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/utils/file_utils.py", line 175, in cached_path
    output_path = get_from_cache(
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/site-packages/evaluate/utils/file_utils.py", line 431, in get_from_cache
    os.makedirs(cache_dir, exist_ok=True)
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/os.py", line 215, in makedirs
    makedirs(head, exist_ok=exist_ok)
  [Previous line repeated 1 more time]
  File "/storage/weinig/anaconda3/envs/cdl/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/export/home/weinig'
